components/application-broker/internal/broker/server.go

```diff
@@ -25,10 +26,12 @@ type (
 
 	provisioner interface {
 		Provision(ctx context.Context, osbCtx osbContext, req *osb.ProvisionRequest) (*osb.ProvisionResponse, *osb.HTTPStatusCodeError)
+		ProvisionProcess(request RestoreProvisionRequest) error
 	}
 
 	deprovisioner interface {
 		Deprovision(ctx context.Context, osbCtx osbContext, req *osb.DeprovisionRequest) (*osb.DeprovisionResponse, error)
+		DeprovisionProcess(req DeprovisionProcessRequest)
```

Comment:
How about calling these functions `DeprovisionReprocess` and `ProvisionReprocess`?

Reply:
changed